app/view_objects/articles/social_image.rb

```diff
@@ -0,0 +1,63 @@
+module Articles
+  class SocialImage
```

Comment:
This is cool! Helpers being global and hard to use anywhere but in views are weird.

Reply:
Agreed. We have the `draper` gem as one other "view helpers are weird" element, but adding view objects to the app is also a big step in the right direction.